snappiershot/compare.py

```diff
@@ -3,11 +3,13 @@
 from operator import itemgetter
 from typing import Any, Callable, Dict, Iterable, List, Sequence, Set, Tuple
 
+from pint.unit import Unit
```

Comment:
[nit style] move import in above group along with other python packages

Reply:
the file actually gets auto organized by the `isort` pre-commit hook to look like that. i can't change it